concourse/scripts/run_job.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash -eu
+set -o pipefail
+
+SCRIPT_DIR=$(cd "$(dirname "$0")" && pwd)
+PIPE='create-bosh-cloudfoundry'
+JOB=$1
+TEMP="/tmp/concourse-run-job.${RANDOM}"
+trap "rm -f $TEMP" ERR
+
+export TARGET_CONCOURSE=deployer
+# shellcheck disable=SC2091
+$("${SCRIPT_DIR}/environment.sh")
+"${SCRIPT_DIR}/fly_sync_and_login.sh"
+FLY="$FLY_CMD -t ${FLY_TARGET}"
+
+# 2 steps prevent fly reading empty input in case step 1 fails
+${FLY} get-pipeline -p ${PIPE} | "${SCRIPT_DIR}"/unbind_job.rb "${JOB}" >$TEMP
+cat $TEMP | ${FLY} set-pipeline -p ${PIPE} -c=/dev/stdin -n
+
+# shellcheck disable=SC2086
+curl -k https://${CONCOURSE_ATC_USER}:${CONCOURSE_ATC_PASSWORD}@${CONCOURSE_URL:8}/api/v1/pipelines/${PIPE}/jobs/${JOB}/builds -d ''
```

Comment:
Could you split the authentication out to a separate argument so that we don't need to do string manipulation:
```sh
curl -k -u "${CONCOURSE_ATC_USER}:${CONCOURSE_ATC_PASSWORD}" "${CONCOURSE_URL}/api/v1/pipelines/${PIPE}/jobs/${JOB}/builds" -d ''
```